source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,44 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. If no document is found, ``findOne`` returns ``null``. If a
+document is found, ``findOne`` returns that document as an Object.
+The following snippet finds a single document from the ``movies``
+collection:
```

Comment:
- Let's add some info:
  - `findOne` can take a query document
  - `findOne` returns the first document in sort order (or [natural order](https://docs.mongodb.com/manual/reference/glossary/#term-natural-order))
  - Maybe some info on projection too?
- "The following snippet..." should be a new paragraph
- "If no document is found" -> "If no document matches the query"